code/__DEFINES/mobs.dm

```diff
@@ -79,6 +79,11 @@
 ///The limb is snouted
 #define BODYTYPE_SNOUTED (1<<5)
 
+//PARIAH EDIT ADDITION
+///The limb has skrelly bits
+#define BODYTYPE_SKRELL (1<<6)
```

Comment:
What is this for? What is non-humanoid about skrell?

Reply:
This is meant for `update_inv_head()` I believe, so that any hats/helmets with a Skrell version use the right icon file.

Reply:
Its for Skrell versions of hats/helmets, yes.
Kinda doubt that anyone will make any sprites for that though.

Reply:
I will :)

Reply:
Based

Reply:
I believe you should at least do the code\modules\mob\living\carbon\human\human_update_icons.dm and add support there for the skrell heads, just like snout support is there.